src/plugins/discord/fetcher.js

```diff
@@ -0,0 +1,143 @@
+// @flow
+
+import * as Model from "./models";
+import {type Snowflake} from "./models";
+
+type FetchEndpoint = (endpoint: string) => Promise<any>;
+
+/**
+ * Provide the Guild ID to fetch against, and the 'limit'
+ * parameter when fetching GuildMembers, Messages, and Reactions.
+ *
+ */
+type FetchOptions = {|
+  membersLimit: number,
+  messagesLimit: number,
+  reactionsLimit: number,
+|};
+
+export type EndCursor = Snowflake;
+
+export type PageInfo = {|
+  +hasNextPage: boolean,
+  +endCursor: EndCursor | null,
+|};
```

Comment:
Would like to stick with `Snowflake`, rather than alias it to `EndCursor`.

Reply:
Good point; you mentioned that earlier, I didn't quite understand what you meant. Updated.